lib/project_types/script/layers/infrastructure/languages/wasm_task_runner.rb

```diff
@@ -17,6 +17,10 @@ def dependencies_installed?
             true
           end
 
+          def install_dependencies
+            nil
+          end
```

Comment:
I'm not sure this is worth it. We should never hit this path, since this runner's `dependencies_installed?` is always true. However, in the spirit of conforming to the interface, I added it. I wasn't sure if `returning nil`, `noop` or `raise  Error (maybe a Shopify::Bug?)` would be more appropriate.

Reply:
I think `noop` makes more sense to me.  I don't think it should raise an error.